### PR TITLE
Add documentation for camera_view

### PR DESCRIPTION
**Pull request by hunterjm**
**Description:**

Adds documentation to show live camera view in Lovelace.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant-polymer/#3086

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by balloob — DISMISSED** [review]

File: source/_lovelace/picture-elements.markdown [hunk 1]
```diff
@@ -373,6 +373,10 @@ camera_image:
   required: false
   description: A camera entity.
   type: string
+camera_view:
+  required: false
+  description: '"live" will show the live view if `stream` is enabled.  Default: auto.'
```
Comment (klaasnicolaas) [inline source/_lovelace/picture-elements.markdown]: Pherhaps `default: auto` ?

File: source/_lovelace/picture-entity.markdown [hunk 2]
```diff
@@ -30,6 +30,10 @@ camera_image:
   required: false
   description: "Camera `entity_id` to use. (not required if `entity` is already a camera-entity)."
   type: string
+camera_view:
+  required: false
+  description: '"live" will show the live view if `stream` is enabled.  Default: auto.'
```
Comment (klaasnicolaas) [inline source/_lovelace/picture-entity.markdown]: Same

File: source/_lovelace/picture-glance.markdown [hunk 3]
```diff
@@ -38,6 +38,10 @@ camera_image:
   required: false
   description: Camera entity as Background image.
   type: string
+camera_view:
+  required: false
+  description: '"live" will show the live view if `stream` is enabled.  Default: auto.'
```
Comment (klaasnicolaas) [inline source/_lovelace/picture-glance.markdown]: Same

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by balloob** [comment]
damn it @klaasnicolaas , I am trying to ship this thing 😆

**Comment by balloob** [comment]
Fixed it.

**Pull request merged**